atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLock.java

```diff
@@ -302,22 +302,29 @@ private void schemaMutationUnlock(long perOperationNodeId) {
     }
 
     private boolean trySchemaMutationUnlockOnce(long perOperationNodeId) {
-        CASResult result;
         try {
-            result = clientPool.runWithRetry(client -> {
+            return clientPool.runWithRetry(client -> {
                 Column existingColumn = queryExistingLockColumn(client);
+                long existingLockId = getLockIdFromColumn(existingColumn);
+                if (existingLockId == GLOBAL_DDL_LOCK_CLEARED_ID) {
+                    return true;
+                }
 
-                Preconditions.checkState(isValidColumnForLockId(existingColumn, perOperationNodeId),
-                        "Trying to unlock unowned lock");
+                Preconditions.checkState(existingLockId == perOperationNodeId,
+                        "Trying to unlock unowned lock. Expected [%s], but got [%s]",
```

Comment:
It's not clear from this message which of {expected, got} is the one we own. Maybe rephrase as "Trying to unlock lockId=%s but a different lock with lockId=%s is taken out."